.ci_support/osx_64_cgotruego_variant_strcgogo_variant_ver2.3.0.yaml

```diff
@@ -1,9 +1,11 @@
 MACOSX_DEPLOYMENT_TARGET:
-- '10.12'
+- '10.13'
+MACOSX_SDK_VERSION:
+- '10.14'
```

Comment:
Looking at build logs xcode 12 is being used and this is problematic because:

[```Xcode 12 requires an Intel-based Mac running macOS Catalina 10.15.4 or later.``` ](https://developer.apple.com/documentation/xcode-release-notes/xcode-12-release-notes)

So change to 10.15.4?